src/modules/Client/Api/Admin.php

```diff
@@ -247,8 +255,9 @@ public function update($data = [])
         if (($data['currency'] ?? null) && $service->canChangeCurrency($client, $data['currency'] ?? null)) {
             $client->currency = $data['currency'] ?? $client->currency;
         }
-
-        $this->di['events_manager']->fire(['event' => 'onBeforeAdminClientUpdate', 'params' => $data]);
+        $eventDataBefore = $data;
```

Comment:
The `update` function doesn't do anything with passwords, this additional logic doesn't need to be here.